pkg/disk/nodeserver.go

```diff
@@ -885,8 +886,8 @@ func (ns *nodeServer) NodeExpandVolume(ctx context.Context, req *csi.NodeExpandV
 	*csi.NodeExpandVolumeResponse, error) {
 	log.Infof("NodeExpandVolume: node expand volume: %v", req)
 
-	volExpandBytes := int64(req.GetCapacityRange().GetRequiredBytes())
-	requestGB := float64((volExpandBytes + 1024*1024*1024 - 1) / (1024 * 1024 * 1024))
+	volExpandBytes := float64(req.GetCapacityRange().GetRequiredBytes())
+	requestGB := math.Floor(volExpandBytes / float64(1024*1024*1024))
```

Comment:
Do all calculation with int64 and bytes? to avoid any potential issue with floating point precision lost.

Reply:
volExpandBytes variable here is only use to generate requestGB, I think it would be fine.